src/test_mode/readme.md

```diff
@@ -20,7 +20,7 @@ sudo make install
 
 ###follow up:
 verify that the installer has placed libJSBSim.so in /usr/local/lib/.
-if you see an error loading libJSBSim.so*, make sure you add /usr/local/lib to LD_LIBRARY_PATH. If you don't want to modify your environment variables, use "-DBUILD_SHARED_LIB=FALSE" instead and JSBSim will be statically linked at build time.
```

Comment:
Did you mean to remove that note on static linking? I think it is a valid point that @cwalexand brought up.